server/schedule/operator/operator.go

```diff
@@ -54,12 +54,14 @@ type Operator struct {
 }
 
 // NewOperator creates a new operator.
-func NewOperator(desc, brief string, regionID uint64, regionEpoch *metapb.RegionEpoch, kind OpKind, approximateSize int64, steps ...OpStep) *Operator {
+func NewOperator(desc, brief string, regionID uint64, regionEpoch *metapb.RegionEpoch, kind OpKind, approximateSize int64,
```

Comment:
this function has so many params.

Reply:
yes, it should use builder model to refactor it.